static/src/javascripts/projects/common/modules/onward/popular.js

```diff
@@ -44,7 +46,11 @@ define([
     };
 
     MostPopular.prototype.prerender = function () {
-        if (!(config.page.shouldHideAdverts || this.mobileMaximumSlotsReached())) {
+        if (!(
+            config.page.shouldHideAdverts || /* sensitive pages */
```

Comment:
@gklopper - I think it might come up even more once we implement the 'core' experience, too - lots of content will be turned on and off in different circumstances, and that logic is going to be both scattered across the codebase and quite annoying to test everywhere.

My immediate thought would be an immutable map of booleans specifying what's on and off. Immutability would force all the related logic together. We could impose that immutability by exposing the map through a module of getters. Maybe that's overengineering, though (a simple map of bools would be super easy to mock in a test).